packages/insomnia-app/app/ui/redux/modules/__tests__/workspace.test.ts

```diff
@@ -51,7 +58,47 @@ describe('workspace', () => {
     });
 
     it('should create collection', async () => {
-      const store = mockStore();
+      const space = await createSpace();
+      const entities = { spaces: { [space._id]: space } };
+      const global = { activeSpaceId: space._id };
+      const store = mockStore({ entities, global });
+
+      store.dispatch(
+        createWorkspace({
+          scope: WorkspaceScopeKeys.collection,
+        }),
+      );
+      const { title, submitName, defaultValue, onComplete } = getAndClearShowPromptMockArgs();
+      expect(title).toBe('Create New Request Collection');
+      expect(submitName).toBe('Create');
+      expect(defaultValue).toBe('My Collection');
+      const workspaceName = 'name';
+      await onComplete(workspaceName);
+      const workspaces = await models.workspace.all();
+      expect(workspaces).toHaveLength(1);
+      const workspace = workspaces[0];
+      expect(workspace.name).toBe(workspaceName);
+      expect(workspace.scope).toBe(WorkspaceScopeKeys.collection);
+      expect(workspace.parentId).toBe(space._id);
```

Comment:
```suggestion
      expect(workspaces[0]).toMatchObject({
        name: workspaceName,
        scope: WorkspaceScopeKeys.collection,
        parentId: space._id,
      });
```

Perhaps something like this would be good?